### DISCUSSION TITLE
可否拖拽网址文本打开新标签页?

### DISCUSSION THREAD

**Discussion by brace-great** [zimocode/smartup#317] [Q&A] [answered]
我在crxmouse里经常用这个,但他的滚轮手势貌似有问题触发不了所以换这个了.但这貌似又没有拖拽网址文本打开新标签页的功能

**Comment by zimocode** [comment]
就是在新标签中打开链接嘛？

**Comment by zimocode** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/29518677/220018183-d4e42294-4cc3-470b-8d5f-22fee811fb0e.png)

**Reply by brace-great** [reply]
哦我还以为这个只能打开链接,好了,多谢
